packages/styled-components/src/constructors/createGlobalStyle.js

```diff
@@ -19,7 +20,7 @@ export default function createGlobalStyle(
   ...interpolations: Array<Interpolation>
 ) {
   const rules = css(strings, ...interpolations);
-  const styledComponentId = `sc-global-${generateComponentId(JSON.stringify(rules))}`;
+  const styledComponentId = `sc-global-${generateComponentId(stringifyRules(rules))}`;
```

Comment:
There shouldn't be any functions in `rules` because of the `css` utility that's used to flatten the styles: https://github.com/styled-components/styled-components/blob/master/packages/styled-components/src/constructors/css.js